playwright-e2e/tests/full_journey/full-citizen-la-journey-tests.spec.ts

```diff
@@ -0,0 +1,295 @@
+      await laBirthFatherLastKnownAddressPage.clickFindAddressButton();
+      await laBirthFatherLastKnownAddressPage.selectAddressOption('0');
+      await laBirthFatherLastKnownAddressPage.clickSaveAndContinue();
+
+      // Father last date address confirmed page
+      await laBirthFatherLastDateConfirmedAddressPage.fillDayLabel('01');
+      await laBirthFatherLastDateConfirmedAddressPage.fillMonthLabel('01');
+      await laBirthFatherLastDateConfirmedAddressPage.fillYearLabel('1990');
+      await laBirthFatherLastDateConfirmedAddressPage.clickSaveAndContinue();
+
+      // Father documents page page
+      await laBirthFatherServeDocumentsPage.checkYesRadioButton();
+      await laBirthFatherServeDocumentsPage.clickSaveAndContinue();
+
+      // Back to main navigaton page
+      await laChildDetailsPage.clickOtherPersonLink();
+
+      // Other person parental responsibility page
+      await laOtherPersonResponsibilityPage.checkNoRadioButton();
+      await laOtherPersonResponsibilityPage.clickSaveAndContinue();
+
+      // Back to main navigation page
+      await laChildDetailsPage.clickPlacementAndOrderLink();
+
+      // courts order number page
+      await laPlacementOrderNumberPage.fillOrderNumberLabel('abc');
+      await laPlacementOrderNumberPage.clickSaveAndContinue();
+
+      // courts date page
+      await laPlacementDateOfOrderPage.fillDayLabel('01');
+      await laPlacementDateOfOrderPage.fillMonthLabel('01');
+      await laPlacementDateOfOrderPage.fillYearLabel('2020');
+      await laPlacementDateOfOrderPage.clickSaveAndContinue();
+
+      // court orders already in place
+      await laPlacementOrdersInPlacePage.checkNoRadioButton();
+      await laPlacementOrdersInPlacePage.clickSaveAndContinue();
+
+      // Back to main navigation page
+      await laChildDetailsPage.clickSiblingCourtOrderDetailsLink();
+
+      // siblings order page
+      await laChildHasOtherSiblingsPage.checkNoRadioButton();
+      await laChildHasOtherSiblingsPage.clickSaveAndContinue();
+
+      // Back to main navigation page
+      await laChildDetailsPage.clickUploadDocumentLink();
+
+      // upload documents page
+      await laUploadDocumentsPage.checkCannotUploadDocumentsCheckbox();
+      await laUploadDocumentsPage.clickSaveAndContinue();
+
+      // Back to main navigation page
+      await laChildDetailsPage.clickReviewAndSubmitButton();
+
+      // Check you answers page
+      await laCheckYourAnswerPage.clickContinue();
+
+      // statement of truth page
+      await laStatementOfTruthPage.fillFullNameLabel('John Smith');
+      await laStatementOfTruthPage.fillJobTItleLabel('solicitor');
+      await laStatementOfTruthPage.fillLocalAuthorityRepresentationLabel('KCC');
+      await laStatementOfTruthPage.checkTruthStatementCheckbox();
+      await laStatementOfTruthPage.clickConfirm();
+
+      await runAccessibilityScan(makeAxeBuilder, testInfo);
```

Comment:
Just wanted to ask... how many times are the accessibility tests being run and how many times do they need to be run?  (I'm sure this is fine but just wanted to check)

Reply:
They are ran once and only ran at the end of the test.